src/views/DetailedProduct.vue

```diff
@@ -1,49 +1,65 @@
 <script>
-
-import { reactive } from "vue";
-import useListStore from "../stores/listStore";
+import { ref } from 'vue'
+import useListStore from '../stores/listStore'
+import { storeToRefs } from 'pinia'
+import { useRoute } from 'vue-router'
 
 export default {
-    setup() {
-        const listStore = useListStore()
-        let quantity = reactive({ value: 0 });
-        let howMany = reactive({ visibility: false });
+  setup() {
+    const route = useRoute()
+    const listStore = useListStore()
+    const { list } = storeToRefs(listStore)
+
+    const product = list.value.find(
```

Comment:
added this const which does a much simpler job of finding the current selected product in the store list.  Now as a variable, I replaced the large amount of template code where you were doing multiple inline searches for the product using array filter and substrings